cmd/geth/config.go

```diff
@@ -348,6 +352,30 @@ func setDefaultMumbaiGethConfig(ctx *cli.Context, config *gethConfig) {
 	// --pprof is enabled in 'internal/debug/flags.go'
 }
 
+// nolint : wsl
+func setDefaultAmoyGethConfig(ctx *cli.Context, config *gethConfig) {
+	config.Node.P2P.ListenAddr = fmt.Sprintf(":%d", 30303)
+	config.Node.HTTPHost = "0.0.0.0"
```

Comment:
Although this config is not used, 127.0.0.1 is advisable.

Reply:
Duplicated from Mumbai. Maybe this can be taken up in another non-related PR.

Reply:
Makes sense.